src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/src/ReplaceCulture.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    /// <summary>
+    /// Replaces the current culture and UI culture for the test.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public class ReplaceCultureAttribute : BeforeAfterTestAttribute
```

Comment:
As with the other ReplaceCulture file, we are using a different mechanism for that. This file should be removed.